build-system/tasks/visual-diff.js

```diff
@@ -418,19 +388,11 @@ async function generateSnapshots(percy, page, webpages) {
     log('fatal', 'No tests left to run!');
     return;
   } else {
-    log('info', 'Executing', colors.cyan(webpages.length),
-        'visual diff tests for each of', colors.cyan(CONFIGS.join(', ')),
-        'configurations');
+    log('info', 'Executing', colors.cyan(webpages.length), 'visual diff tests');
   }
 
-  for (const config of CONFIGS) {
-    applyAmpConfig(config);
```

Comment:
If we remove this, don't we need to apply the prod config outside?

Reply:
This is redundant. It's already done by default during the build.